packages/discount-code-generator/readme.md

```diff
@@ -0,0 +1,11 @@
+# discount-code-generator
+
+Generate discount codes to be imported to the commercetools platform.
+
+https://commercetools.github.io/nodejs/docs/cli/discount-code-generator.h
```

Comment:
Url should be something like this: `https://commercetools.github.io/nodejs/cli/csv-parser-price.html`